keyboards/keebio/bdn9/keymaps/bcat/keymap.c

```diff
@@ -0,0 +1,43 @@
+#include QMK_KEYBOARD_H
+
+enum layer {
+    LAYER_FIRST,
+    LAYER_SECOND,
+};
+
+/* Switch to second layer when held. */
+#define LY_SECND MO(LAYER_SECOND)
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* First layer (F1-F6) */
+    [LAYER_FIRST] = LAYOUT(
+        KC_MUTE,  LY_SECND, BL_TOGG,  \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".

Reply:
Ah yeah, good point. I... have no idea why I put these here, actually. I think I blindly copied them from a macro definition (where they are needed) into my keymaps. Whoops.

This PR is already merged, but I'll send followup to remove the spurious backslashes.

Reply:
Done in #5927